gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/ComplianceEvents.java

```diff
@@ -38,4 +38,11 @@
   public static class Restore {
     public static final String FAILED_EVENT_NAME = "RestoreFailed";
   }
+
+  public static class Purger {
+    public static final String WORKUNIT_GENERATED = "WorkUnitGenerated";
+    public static final String WORKUNIT_COMMITTED = "WorkUnitCommitted";
+    public static final String WORKUNIT_FAILED = "WorkUnitFailed";
+    public static final String CYCLE_COMPLETED = "CycleCompleted";
```

Comment:
Would be better to qualify it as PurgerCycleCompleted, to identify which  cycle the event is referring to.